hikaricp/tests/test_hikaricp.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from datadog_checks.dev.utils import get_metadata_metrics
+
+EXPECTED_METRICS = {
+    "hikaricp.connections",
+    "hikaricp.connections.active",
+    "hikaricp.connections.idle",
+    "hikaricp.connections.pending",
+    "hikaricp.connections.timeout.count",
+    "hikaricp.connections.acquire.seconds.count",
+    "hikaricp.connections.acquire.seconds.max",
+    "hikaricp.connections.acquire.seconds.sum",
+    "hikaricp.connections.creation.seconds.count",
+    "hikaricp.connections.creation.seconds.sum",
+    "hikaricp.connections.creation.seconds.max",
+    "hikaricp.connections.usage.seconds.max",
+    "hikaricp.connections.usage.seconds.count",
+    "hikaricp.connections.usage.seconds.sum",
+}
+
```

Comment:
In tests, is it possible to check for tags ?
(example: `pool` which is _the_ important tag here)